pkg/sidecar/sidecar.go

```diff
@@ -229,6 +231,19 @@ func (s *Sidecar) dumpBundles(svidResponse *workloadapi.X509Context) error {
 		certs = []*x509.Certificate{certs[0]}
 	}
 
+	// If using federated domains, add them to the CA bundle
+	if s.config.FederatedTrustDomains {
```

Comment:
can you add a unit test to validate this?

Reply:
I just included it